tests/Utils/SchemaPrinterTest.php

```diff
@@ -420,6 +420,37 @@ public function testPrintsStringFieldWithMultipleArgsLastIsDefault(): void
         );
     }
 
+    /**
+     * @see it('Prints Fields With Description Args')
+     */
+    public function testPrintsFIeldsWithDescriptionArgs(): void
+    {
+        $schema = $this->buildSingleFieldSchema([
+            'type' => Type::string(),
+            'args' => [
+                'argOne' => ['type' => Type::int(), 'description' => 'This is the first argument'],
+                'argTwo' => ['type' => Type::string(), 'description' => 'This is the second argument'],
+                'argThree' => ['type' => Type::boolean()],
+            ],
+        ]);
+        self::assertPrintedSchemaEquals(
+            <<<'GRAPHQL'
+            type Query {
+              singleField(
+                """This is the first argument"""
+                argOne: Int
+
+                """This is the second argument"""
+                argTwo: String
+                argThree: Boolean
```

Comment:
Personally, I would find it nicer if there was an extra newline after the arg that has a description.
```suggestion
                """This is the second argument"""
                argTwo: String

                argThree: Boolean
```

Reply:
Me too! Would you like to make that change in this PR? I am fine with diverging from the reference implementation here.

Reply:
I'll try it in a new PR :)

Reply:
Done in https://github.com/webonyx/graphql-php/pull/1141